target/linux/ath79/dts/ar9344_pcs_cap324.dts

```diff
@@ -103,10 +103,26 @@
 				reg = <0x050000 0x0fa0000>;
 			};
 
-			art: partition@7f0000 {
+			partition@7f0000 {
 				label = "art";
 				reg = <0xff0000 0x010000>;
 				read-only;
+
+				compatible = "nvmem-cells";
+				#address-cells = <1>;
+				#size-cells = <1>;
+
+				macaddr_art_0: macaddr@0 {
+					reg = <0x0 0x6>;
+				};
+
+				calibration_art_1000: calibration@1000 {
+					reg = <0x1000 0x1000>;
+				};
+
+				calibration_art_5000: calibration@5000 {
+					reg = <0x5000 0x1000>;
+				};
```

Comment:
Not sure if the length `0x1000` is here correct.

Reply:
I remember that no device uses an eeprom with a length of `0x1000` in ath9k platform. They should be `0x440` or `0xeb8`.

Reply:
But that's what currently extracted by some platforms? `caldata_extract "art" 0x1000 0x1000`

Reply:
It seems that they are wrong. For example, the wnr2200 seems to be `0x3d8` https://github.com/openwrt/openwrt/pull/10967#issuecomment-1326418923

Reply:
Here is an art collection. The same chip should have the same art size.  [art.zip](https://github.com/openwrt/openwrt/files/10259576/art.zip)

Reply:
Nice. So let's change the wrong ones?

Reply:
> It seems that they are wrong. For example, the wnr2200 seems to be `0x3d8` [#10967 (comment)](https://github.com/openwrt/openwrt/pull/10967#issuecomment-1326418923)

Why `0x3d8`? I thought it is `0x3e0`?

Reply:
> Why `0x3d8`? I thought it is `0x3e0`?

I remember that the eeprom of the ath9k has a crc check. But I'm not sure about the exact length.

Reply:
I have confirmed on an **AR9287** device that `0x3d8` is the correct size. Any value less than `0x3d8` will cause the initialization of ath9k to fail. `0x3e0` should still be okay.
```
Fri Jan 20 21:57:19 2023 kern.err kernel: [   18.025901] ath9k 0000:00:00.0: Failed to initialize device
Fri Jan 20 21:57:19 2023 kern.warn kernel: [   18.031591] ath9k: probe of 0000:00:00.0 failed with error -22
```